collection-spec/collection-spec.md

```diff
@@ -270,15 +270,16 @@ It is recommended to use the official
-| child        | URL to a child STAC entity (Catalog or Collection).                                                                                                                                                                                                                |
-| item         | URL to a STAC Item. All Items linked from a Collection MUST refer back to its Collection with the [`collection` relation type](../item-spec/item-spec.md#relation-types).                                                                                          |
-| license      | The license URL(s) for the Item SHOULD be specified if the `license` field is **not** a SPDX license identifier.                                                                                                                                                   |
-| derived_from | URL to a STAC Collection that was used as input data in the creation of this Collection. See the note in [STAC Item](../item-spec/item-spec.md#derived_from) for more info.                                                                                        |
+| Type         | Description                                                                                                                                                                                                                                                                         |
+| ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| self         | STRONGLY RECOMMENDED. *Absolute* URL to the location that the Collection file can be found online, if available. This is particularly useful when in a download package that includes metadata, so that the downstream user can know where the data has come from.                  |
+| root         | URL to the root STAC entity (Catalog or Collection). Collections should include a link to their root, even if it's the root and points to itself.                                                                                                                                   |
+| parent       | URL to the parent STAC entity (Catalog or Collection). Non-root Collections should include a link to their parent.                                                                                                                                                                  |
+| collection   | URL to a parent Collection. *Absolute* URLs should be used whenever possible. The referenced Collection is STRONGLY RECOMMENDED to implement the same STAC version as the Collection. A link with this `rel` type is *required* if the `collection` field in properties is present. |
```

Comment:
I think the changelog should mention that we add the collection relation type for collections.
It can point to issue https://github.com/radiantearth/stac-spec/issues/1236